cosmwasm-plugin/main.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"embed"
+	"encoding/gob"
+	"fmt"
+	"io"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	hplugin "github.com/hashicorp/go-plugin"
+
+	"github.com/ignite/cli/ignite/services/chain"
+	"github.com/ignite/cli/ignite/services/plugin"
+)
+
+var (
+	chainName  string
+	chainNamed string
+	//go:embed wasm-wiring/*
+	templates embed.FS // Embedded template files
+
+)
```

Comment:
```suggestion
var (
	chainName  string
	chainNamed string
	
	//go:embed wasm-wiring/*
	templates embed.FS // Embedded template files
)
```

Reply:
Removed global variables in this [commit](https://github.com/ignite/plugins/commit/38af11e00e770736f6f3a9bfa37d04dc241bd865)